cmd/cosign/cli/attest/attest.go

```diff
@@ -116,15 +117,20 @@ func AttestCmd(ctx context.Context, ko options.KeyOpts, regOpts options.Registry
 	wrapped := dsse.WrapSigner(sv, types.IntotoPayloadType)
 	dd := cremote.NewDupeDetector(sv)
 
+	var predicate []byte
 	fmt.Fprintln(os.Stderr, "Using payload from:", predicatePath)
```

Comment:
I think this `fmt.Fprintln` should go in the "else" case below: "Using payload from: -" is a little weird as a message. You *could* add a message for "Using payload from: standard input" or something, but I don't think it's necessary.

Reply:
absolutely!